spec/features/user_views_past_orders_spec.rb

```diff
@@ -16,14 +12,43 @@
   end
 
   it "has an orders page for user" do
+    login_user(user)
+    visit '/profile'
+    click_link "View Past Orders"
+
+    expect(current_path).to eq orders_path
+  end
+
+  it "has a link to an individual order page" do
+    login_user(user)
+    create_order(user)
+
+    visit orders_path
+
+    within ".order-table tbody tr:first" do
+      click_link "View Order"
+    end
+
+    expect(current_path).to eq order_path
+  end
+
+  xit "order page displays order detail" do
+  end
+
+  def login_user(user)
     visit '/login'
     fill_in "Username", with: user.username
     fill_in "Password", with: user.password
     click_button "Login"
+  end
 
-    visit '/profile'
-    click_link "View Past Orders"
-
-    expect(current_path).to eq orders_path
+  def create_order(user)
+    d     = Destination.create(continent: "North America", description: "North American continent description.",
+                        lat: 41.8369, long: -87.6847, image_url: "south-america.jpg" )
```

Comment:
Line is too long. [86/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.